components/upvote.js

```diff
@@ -67,11 +67,20 @@ export function DropdownItemUpVote ({ item }) {
   )
 }
 
-export const nextTip = (meSats, { tipDefault, turboTipping }) => {
+export const defaultTipIncludingRandom = ({ tipDefault, tipRandom, tipRandomMin, tipRandomMax }) =>
```

Comment:
A reusable function for determining the next zap value that honors `tipRandom` and `tipDefault`